app/models/support_interface/applications_filter.rb

```diff
@@ -196,6 +196,11 @@ def status_filter
             label: 'Offer deferred',
             checked: applied_filters[:status]&.include?('offer_deferred'),
           },
+          {
+            value: 'recruited',
```

Comment:
could we remove the conditions met?

Reply:
Sorry for the delay, that's done - do you know if there's an appropriate position for `Recruited` to appear on the list

Reply:
I think it should appear in place of `conditions_met` as it seems the list is ordered regarding decision

Reply:
Fixed, thanks!